zephyr/include/sof/lib/dma.h

```diff
@@ -193,36 +177,25 @@ struct dma_chan_status {
 	void *ipc_posn_data;
 };
 
-/* DMA operations */
-struct dma_ops {
-
-	struct dma_chan_data *(*channel_get)(struct dma *dma,
-					     unsigned int req_channel);
-	void (*channel_put)(struct dma_chan_data *channel);
-
-	int (*start)(struct dma_chan_data *channel);
-	int (*stop)(struct dma_chan_data *channel);
-	int (*stop_delayed)(struct dma_chan_data *channel);
-	int (*copy)(struct dma_chan_data *channel, int bytes, uint32_t flags);
-	int (*pause)(struct dma_chan_data *channel);
-	int (*release)(struct dma_chan_data *channel);
-	int (*status)(struct dma_chan_data *channel,
-		      struct dma_chan_status *status, uint8_t direction);
-
-	int (*set_config)(struct dma_chan_data *channel,
-			  struct dma_sg_config *config);
-
-	int (*probe)(struct dma *dma);
-	int (*remove)(struct dma *dma);
+struct dma_chan_data {
```

Comment:
not a reason for a new revision, but **if** you do end up doing another update - would it be possible to keep this structure at the same place as now to reduce the diff?

Reply:
@lyakh Ack, fixed in new revision.